app-i18n/libime/metadata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>tanekliang@gmail.com</email>
+		<name>Yongxiang Liang</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="github">fcitx/libime</remote-id>
+	</upstream>
+	<use>
+		<flag name="coverage">Build the project with gcov support</flag>
```

Comment:
Is running coverage in tests useful in the ebuild? This would be something that upstream should be interested but running it downstream seems unnecessary.